geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -3101,7 +3101,7 @@ boolean cacheWriteBeforeRegionDestroy(RegionEventImpl event)
     return result;
   }
 
-  private void cacheWriteBeforeRegionClear(RegionEventImpl event)
+  protected void cacheWriteBeforeRegionClear(RegionEventImpl event)
```

Comment:
Do you really need to increase the visibility here?

Reply:
fixed